src/adapter/etl-adapter-http/tests/Flow/ETL/Tests/Functional/PsrHttpClientStaticExtractorTest.php

```diff
@@ -8,18 +8,25 @@
 use Flow\ETL\Config;
 use Flow\ETL\FlowContext;
 use Flow\ETL\Rows;
-use Http\Client\Curl\Client;
+use Http\Mock\Client;
 use Nyholm\Psr7\Factory\Psr17Factory;
+use Nyholm\Psr7\Response;
 use PHPUnit\Framework\TestCase;
 
 final class PsrHttpClientStaticExtractorTest extends TestCase
 {
     public function test_http_extractor() : void
     {
         $psr17Factory = new Psr17Factory();
-        $psr18Client = new Client($psr17Factory, $psr17Factory);
-
-        $requests = function () use ($psr17Factory) : \Generator {
+        $psr18Client = new Client($psr17Factory);
+        $psr18Client->addResponse(
```

Comment:
those are no longer functional tests, I would probably call then integration but since they were failing way too frequently it's better to keep them that way 👍